content/docs/user-guide/gems/reference/kythera-ai/spatial-query-system.md

```diff
@@ -0,0 +1,723 @@
+            
+        *   ShipRadius: float (default 0.0) special value -1.0 means "Use querier's radius"
+            
+        *   NumPoints: int (default 25) Number of points to try and return
+            
+        *   PathDistanceRatio: float (default 1.5) This times MaxRadius is used as maxPathDistance
+            
+        *   ConeAngle: float (default 0.0) Width of cone to limit results. 0.0 special value means whole sphere
+            
+        *   ConeDirection: Vec3 (default 0,0,0) Direction of centre of cone, if ConeAngle is non-zero
+            
+        *   AvoidDynamicObstacles: bool (default false)
+            
+        *   OpenAreasOnly: bool (default false) True can speed up results but skips points close to unnavigable regions
+            
+        *   PointRadius: float (default 0.0) Causes points too close to each other to be removed
+            
+
+#### `OctreeGridNavigableGenerator`
+
+*   Generates points in a grid within a sphere, spherical shell, or a cone which are navigable according to the octree
+    
+    *   Named parameters:
+        
+        *   Origin: Querier, Reference or Target
+            
+        *   MinRadius: float (default 3.0)
+            
+        *   MaxRadius: float (default 4.0)
+            
+        *   ShipRadius: float (default 0.0) special value -1.0 means "Use querier's radius"
+            
+        *   Density: float (default 1.0) - spacing of grid
+            
+        *   PathDistanceRatio: float (default 1.5) This times MaxRadius is used as maxPathDistance
+            
+        *   ConeAngle: float (default 0.0) Width of cone to limit results. 0.0 special value means whole sphere
+            
+        *   ConeDirection: Vec3 (default 0,0,0) Direction of centre of cone, if ConeAngle is non-zero
+            
+        *   AvoidDynamicObstacles: bool (default false)
+            
+        *   OpenAreasOnly: bool (default false) True can speed up results but skips points close to unnavigable regions
+            
+        *   PointRadius: float (default 0.0) Causes points too close to each other to be removed
+            
+
+* * *
+
+### Logical functions  
+
+#### Direct logical operations
+
+*   `LessThan(lhs, rhs)`
+    
+*   `LessThanOrEqualTo(lhs, rhs)`
+    
+*   `GreaterThan(lhs, rhs)`
+    
+*   `GreaterThanOrEqualTo(lhs, rhs)`
+    
+*   `Equal(lhs, rhs)`
+    
+*   `NotEqual(lhs, rhs)`
+    
+*   `Not(boolean)`
+    
+
+#### Grouped logical operations
+
+*   `AllOf(expressions...)`
+    
+    *   Returns true if every expression is true. Will return early if any expression returns false.
+        
+*   `AnyOf(expressions...)`
+    
+    *   Returns true if any single expression returns true. Will return early if any expression returns true.
+        
+*   `NoneOf(expressions...)`
+    
+    *   Returns true if none of the expressions return true. Will return early if any expression returns true.
+        
+
+* * *
+
+### Operation functions
```

Comment:
I'm not sure if the bold formatting was meant to double as a grouping for categorization purposes, but if it was, it's not going to be obvious to everyone. Therefore I've replaced all the bold with code formatting. If you want to categorize, I recommend explicitly inserting category labels. If you don't want to categorize, then I'd recommend alphabetizing the whole list.